src/app/App.spec.ts

```diff
@@ -17,6 +17,8 @@ const router = createRouter({
 })
 
 function renderComponent(status: string) {
+  store.state.globalLoading = true
+  store.state.config.tagline = 'Kuma'
```

Comment:
I _could_ somehow read the .env file for the tagline to make this project-agnostic, but since this is a test, there is no impact on UI. But perhaps it’s the better strategy for avoiding conflicts.

Reply:
Made some adjustments to make this more robust.